tutorials/index.md

```diff
@@ -4,6 +4,15 @@ title = "Tutorials"
 
 # Tutorials
 
+## 2023 "Julia for HPC", Oak Ridge Leardership Facility (OLCF) 
+
+1. Event [website](https://www.olcf.ornl.gov/calendar/julia-for-high-performance-computing-tutorial/)
+2. Introduction [slides](https://drive.google.com/file/d/119EnUy-psHICmiZompWa1ZxOo9JsTq4Y/view?usp=drive_link)
+3. Tutorial [website](https://juliaornl.github.io/TutorialJuliaHPC/)
+4. Gray-Scott application on [GitHub](https://github.com/JuliaORNL/GrayScott.jl)
+5. [Scripts](https://github.com/JuliaORNL/GrayScott.jl/tree/main/scripts) for running for OLCF systems: Summit, Ascent, Crusher
+6. Recording hosted on [Vimeo](https://vimeo.com/830368460?share=copy)
```

Comment:
The video requires logging into Vimeo.  Can it be made public?

Reply:
@giordano it should be public, I think it's just Vimeo asking to join/log in.

Reply:
Sounds like the problem is that the video is not rated?  It should probably be easy for whoever uploaded the video to fix this.

Reply:
I can see it without logging in. Let me ask around

Reply:
![Screenshot_20230608_003502](https://github.com/JuliaParallel/juliaparallel.github.io/assets/765740/c5813d8c-086a-47c1-a5a0-3a158c6074a6)
I see the same on different devices.

Reply:
Thanks! I will check with OLCF. Maybe it depends on country up address restrictions?

Reply:
Maybe it is geofenced?

Reply:
But only the Julia video? I can watch the other videos of the same playlist. @sloede can you watch it?

Reply:
> But only the Julia video?

Ah, you're right, that doesn't make sense. And no, I can't watch it either

Reply:
Please try again and let me know. Hopefully it should work now.

Reply:
Yes, it does, thank you!